### DISCUSSION TITLE
如何自定义 Swagger 文档里的示例数据

### DISCUSSION THREAD

**Discussion by wangzhatian** [apiflask/apiflask#153] [Q&A] [answered]
[截图](https://imgfigure1.oss-cn-hangzhou.aliyuncs.com/20211020180344.png)

**Comment by greyli** [comment, marked as answer]
参考这里自定义示例的值：https://apiflask.com/openapi/#response-and-request-example

**Reply by greyli** [reply]
@wangzhatian 后续如果有中文问题请发到[论坛](https://discuss.helloflask.com)。
